datasafe-rest/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM openjdk:8-jre-alpine
+ENV JAR_FILE datasafe-rest-0.0.6-SNAPSHOT.jar
```

Comment:
Hardcoded to release number, maybe we can drop release number and use wildcard?
I.e. copy datasafe-rest-*.jar or use variables coming from maven:
https://vsupalov.com/docker-build-pass-environment-variables/